src/Apps/Artwork/Components/ArtworkBanner/__tests__/ArtworkBanner.test.tsx

```diff
@@ -53,6 +54,24 @@ describe("ArtworkBanner", () => {
     })
   })
 
+  describe("ArtworkBanner for artwork with benefit auction banner", () => {
+    beforeAll(async () => {
+      wrapper = await getWrapper(ArtworkBenefitAuctionBannerFixture)
+    })
+    it("renders a correct data for the auction", () => {
+      const html = wrapper.html()
+
+      expect(html).toContain("In auction")
+      // expect(html).toContain("BFAMI: Live Benefit Auction 2019")
```

Comment:
stray comment

Reply:
It's actually not stray. I need to add #TODO: here as well as the other test. The issue for some reason the data from the mocked artwork is not coming through if the same property name is shared for different contexts. So neither 'name' or 'href' are mocked properly for this auction banner.

@alloy and myself tried to look at it at some point but did not get to a resolution. So this line even though theoretically should be true would fail in this test right now:(

Reply:
Cool cool, a `TODO` pointing to that comment would be good 👍